claudia-lab/app/config/router-config.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider){
+  $urlRouterProvider.when('/', '/home');
+  $urlRouterProvider.when('/hello', '/home');
+
+  let routes = [
+    {
+      name: 'home',
+      url: '/home',
+      template: require('../view/home/home.html'),
+      controller: 'HomeController',
+      controllerAs: 'homeCtrl',
+    },
+    {
+      name: 'signup',
+      url: '/signup',
+      template: require('../view/signup/signup.html'),
+      controller: 'SignupController',
+      controllerAs: 'signupCtrl',
+    },
+    {
+      name: 'profile',
+      url: '/profile',
+      template: require('../view/profile/profile.html'),
+      controller: 'ProfileController',
+      controllerAs: 'profileCtrl',
+    },
+  ];
+
+  routes.forEach( route => {
+    $stateProvider.state(route);
+  });
+}
+
+
+// view
```

Comment:
@loomnugget this comment block is fine for showing the tree but is best contained on a separate branch, in order to keep `master` production ready